plugin.yml

```
@@ -102,3 +104,4 @@ configuration:
     user: [ run ]
     propagate-uid-gid: [ run ]
     mount-buildkite-agent: [ run ]
+    use-prior-image: { build }
```

I believe that, at least for consistency, these should be square brackets:
```suggestion
    use-prior-image: [ build ]
```

---

Thanks! I'm not sure if these changed, I've been rebasing this PR from time to time and just layering the changes onto what was upstream.

---

From what I can see in the history of the repository, the square brackes have been used as early as 5 years ago :shrug:

---

Lo and behold, a second version of this on another branch did this the right way. No idea how this snuck in, but fixed now.